azure-pipelines.yml

```diff
@@ -256,14 +248,6 @@ steps:
       flattenFolders: true
     displayName: 'Copy JS Content for NPM'
 
-  - task: CopyFiles@2
```

Comment:
We dont need this script in release pipeline as, as we are using the Out of the Box task in release pipeline to publish